core/trie/trie.go

```diff
@@ -38,14 +38,26 @@ type Trie struct {
 	height  uint
 	rootKey *bitset.BitSet
 	storage Storage
+	hash    func(*felt.Felt, *felt.Felt) *felt.Felt
 }
 
+type NewTrieFunc func(Storage, uint, *bitset.BitSet) *Trie
+
 func NewTrie(storage Storage, height uint, rootKey *bitset.BitSet) *Trie {
```

Comment:
Please rename `NewTrie` to `NewTriePedersen`

Reply:
done